scripts/publish_packages.sh

```diff
@@ -56,6 +56,10 @@ if [[ "${TRAVIS_PUBLISH_PACKAGES:-}" == "true" ]]; then
         --skip-existing \
         --verbose
 
+    echo "Publishing .nupkgs to nuget.org:"
+    find ~/.nuget/local/ -name '*.nupkg' \
```

Comment:
Note that this doesn't seem very safe if this is ever done from a local machine.

Reply:
user would have ot have both TRAVIS_PUBLISH_PACKAGES set *and* NUGET_PUBLISH_KEY set.  so i'm not really concerned about htat.

Reply:
We might need to do this from a local machine ourselves, e.g. to manually publish packages.

Reply:
Updated this to ensure that build cleans this directory and then copies the built artifacts into it.  so in hte manual case, it should only be what was built last.